instrumentation/micronaut-1.0/build.gradle.kts

```diff
@@ -27,11 +27,8 @@ for (version in listOf(micronaut2Version)) {
         extendsFrom(configurations.testRuntimeClasspath.get())
     }
     dependencies {
-        versionedConfiguration(project(":instrumentation:netty:netty-4.1"))
-        versionedConfiguration("io.opentelemetry.javaagent.instrumentation:opentelemetry-javaagent-netty-4.1:${versions["opentelemetry_java_agent"]}")
         versionedConfiguration(project(":testing-common"))
         versionedConfiguration("io.micronaut.test:micronaut-test-junit5:${micronautTestVersion}")
-        versionedConfiguration("io.micronaut:micronaut-inject-java:${version}")
         versionedConfiguration("io.micronaut:micronaut-http-server-netty:${version}")
         versionedConfiguration("io.micronaut:micronaut-http-client:${version}")
         versionedConfiguration("io.micronaut:micronaut-runtime:${version}")
```

Comment:
Doesn't let me comment, but line 45 should be removed. 47 may or may not change depending on your response to https://github.com/hypertrace/javaagent/pull/232#discussion_r556675481

Reply:
45 removed 👍